### PR TITLE
advise(react/errorboundary): add & change example case

### PR DESCRIPTION
**Pull request by minsoo-web**
# Overview

This PR is for clear example case for #216 

## PR Checklist

- [x] I have written documents and tests, if needed.

### CODE REVIEW

File: packages/react/src/ErrorBoundary.ko.mdx [hunk 1]
```diff
@@ -50,27 +50,57 @@ const ErrorAfter4s = () => {
...
+:::
+
+:::tip
+
 ## `<ErrorBoundary/>` fallback props을 prop drilling 없이 사용하기
 
 ### useErrorBoundaryFallbackProps
 
-ErrorBoundaryFallback에서 useErrorBoundaryFallbackProps().reset/error을 사용해 prop을 직접 사용하지 않고도 `<ErrorBoundary/>`을 reset하고 error를 사용할 수 있습니다.
+만약, fallback 컴포넌트의 depth가 깊어진다면, `error` 객체와 `reset` 메소드에 접근하기 위해 prop drilling을 피할 수 없을 것 입니다.
```
Comment (manudeli) [inline packages/react/src/ErrorBoundary.ko.mdx:84]:
```suggestion
`error` 객체와 `reset` 메소드을 사용하려는 컴포넌트가 중첩되면 prop drilling을 피할 수 없습니다.
```

File: packages/react/src/ErrorBoundary.en.mdx [hunk 2]
```diff
@@ -50,27 +50,57 @@ const ErrorAfter4s = () => {
...
+:::
+
+:::tip
+
 ## Use `<ErrorBoundary/>` fallback props without prop drilling
 
 ### useErrorBoundaryFallbackProps
 
-In fallback of `<ErrorBoundary/>`, by calling useErrorBoundaryFallbackProps().reset/error, we can use error of `<ErrorBoundary/>` and reset it without using prop directly
+If the depth of the fallback component deepens, prop drilling will not be avoided to access `error` objects and `reset` methods.
```
Comment (manudeli) [inline packages/react/src/ErrorBoundary.en.mdx:84]:
```suggestion
If component using `reset` method and `error` object is nested, prop drilling cannot be avoided.
```

**Review by manudeli — COMMENTED** [review]
I left comment to edit some sentences as simple, commit suggestion please if you okay @minsoo-web

**Comment by minsoo-web** [comment]
> I left comment to edit some sentences as simple, commit suggestion please if you okay @minsoo-web 

It is applied, thanks !!

**Review by manudeli — APPROVED** [review]

**Comment by manudeli** [comment]
Better examples thanks!

**Pull request merged**